.github/workflows/apcd-cms.yml

```
@@ -36,4 +37,25 @@ jobs:
         with:
           context: apcd_cms
           push: true
-          tags: taccwma/apcd-cms:${{ steps.vars.outputs.SHORT_SHA }},taccwma/apcd-cms:${{ steps.vars.outputs.BRANCH_NAME }}
\ No newline at end of file
+          tags: taccwma/apcd-cms:${{ steps.vars.outputs.SHORT_SHA }},taccwma/apcd-cms:${{ steps.vars.outputs.BRANCH_NAME }}
+  sync:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: apcd_cms
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with: 
+          fetch-depth: 0
+
+      - name: Update develop-apcd Branch when changes to main are made
+        run: |
+          git checkout main
+          git fetch origin
+          git checkout develop-apcd
+          git pull
+          git merge origin/main
+          git push origin develop-apcd
```

Wouldn't this be ephemeral every time? Or is there a persistent cache for git - if not, we might be able to simplify it
```suggestion
          git checkout develop-apcd
          git merge origin/main
          git push origin develop-apcd
```